### PR TITLE
Coledishington/flow force reuse 6293 v7

### PR DESCRIPTION
**Pull request by catenacyber**
Add test for verifying that if the memcap is reached, alive flows will be reused with flow.force-reuse on and not be re-used with flow.force-reuse off.

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/6293

https://github.com/OISF/suricata-verify/pull/1699 with additional commit to run only on arch x86_64

### CODE REVIEW

File: tests/flow-force-reuse/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Test Purpose
+
+Tests Suricata with the configuration option flow.force-reuse on and
+off. This is done by having a low flow memcap and long-lived
+connections.
+
+## PCAP
+
+This PCAP was generated with scapy.
```
Comment (inashivb) [inline tests/flow-force-reuse/README.md:9]: Could we have the ticket number here?

File: tests/flow-force-reuse/flow-force-reuse-off/test.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 8
+  arch: x86_64
+
+pcap: ../test.pcap
+
+args:
+- --set flow.memcap-policy=drop-flow
```
Comment (inashivb) [inline tests/flow-force-reuse/flow-force-reuse-off/test.yaml:8]: does this matter in this case?

**Review by inashivb — COMMENTED** [review]
Looking good. Minor comments inline

File: tests/flow-force-reuse/README.md (hunk 1 above)
Comment (catenacyber) [reply tests/flow-force-reuse/README.md]: added

**Comment by catenacyber** [comment]
@coledishington closing here as Suricata PR, waiting for your PR

**Pull request closed without merging**